superflore/utils.py

```diff
@@ -170,9 +170,12 @@ def get_license(l):
             return 'Apache-%.1f' % (float(version))
         return 'Apache-1.0'
     elif re.search(bsd_re, l, f):
-        version = re.search(bsd_re, l, f).group(4)
+        matches = re.search(bsd_re, l, f)
+        version = matches.group(4)
+        if not version:
+            version = matches.group(5)
```

Comment:
@allenh1  And here I just try to get the version from either the previous kind of match, or the new (giving priority to the previous case).